models/arkham/ethereum/arkham_ethereum_airdrop_claims.sql

```diff
@@ -0,0 +1,46 @@
+{{
+    config(
+        tags=['dunesql'],
+        schema = 'arkham_ethereum',
```

Comment:
Maybe we do not need to add the new legacy spell only when the schema is already existed in any legacy spells.